internal/monitors/logstash/logstash/metadata.yaml

```diff
@@ -0,0 +1,427 @@
+      default: true
+      type: cumulative
+      group: pipeline
+    node.stats.pipelines.events.in:
+      description: Number of incoming events to pipelines
+      default: true
+      type: cumulative
+      group: pipeline
+    node.stats.pipelines.events.out:
+      description: Number of outgoing events from pipelines
+      default: true
+      type: cumulative
+      group: pipeline
+    node.stats.pipelines.events.queue_push_duration_in_millis:
+      description: Queue push duration of events
+      default: false
+      type: gauge
+      group: pipeline
+    node.stats.pipelines.plugins.codecs.decode.duration_in_millis:
+      description: Duration of decode events in codec plugins
+      default: false
+      type: gauge
+      group: pipeline
+    node.stats.pipelines.plugins.codecs.decode.out:
+      description: Number of outgoing decode events from codecs
+      default: false
+      type: cumulative
+      group: pipeline
+    node.stats.pipelines.plugins.codecs.decode.writes_in:
+      description: Number of incoming decode events to codecs
+      default: false
+      type: cumulative
+      group: pipeline
+    node.stats.pipelines.plugins.codecs.encode.duration_in_millis:
+      description: Duration of encode events in codec plugins
+      default: false
+      type: gauge
+      group: pipeline
+    node.stats.pipelines.plugins.codecs.encode.writes_in:
+      description: Number of incoming encode events to codecs
+      default: false
+      type: cumulative
+      group: pipeline
+    node.stats.pipelines.plugins.filters.events.duration_in_millis:
+      description: Duration of events in filter plugins
+      default: true
+      type: gauge
+      group: pipeline
+    node.stats.pipelines.plugins.filters.events.in:
+      description: Number of incoming events to filters
+      default: true
+      type: cumulative
+      group: pipeline
+    node.stats.pipelines.plugins.filters.events.out:
+      description: Number of outgoing events from filters
+      default: true
+      type: cumulative
+      group: pipeline
+    node.stats.pipelines.plugins.inputs.events.out:
+      description: Number of outgoing events from inputs
+      default: true
+      type: cumulative
+      group: pipeline
+    node.stats.pipelines.plugins.inputs.events.queue_push_duration_in_millis:
+      description: Queue push duration of events in input plugins
+      default: false
+      type: gauge
+      group: pipeline
+    node.stats.pipelines.plugins.outputs.events.duration_in_millis:
+      description: Duration of events in output plugins
+      default: true
+      type: gauge
+      group: pipeline
+    node.stats.pipelines.plugins.outputs.events.in:
+      description: Number of incoming events to output plugins
+      default: true
+      type: cumulative
+      group: pipeline
+    node.stats.pipelines.plugins.outputs.events.out:
+      description: Number of outgoing events from output plugins
```

Comment:
This description is somewhat misleading.  The way it is phrased it sounds like a gauge ("outgoing" is present participle, and gauges represent present state), not a cumulative counter.